assets/components/pages/research-detail/research-detail.twig

```diff
@@ -0,0 +1,88 @@
+{% extends "@templates/base/base.twig" %}
+
+{% block grid %}{% endblock %}
```

Comment:
Seems odd to me 🤔  Shouldn't we use the same approach as the lab page ?

Reply:
Not necessarly as this is designed for a tool that is not a CMS, and that has no user-inputted content, we can forgo the container-grid that assures text to be displayed in a good width